lib/request/sfWebRequest.class.php

```diff
@@ -75,7 +75,14 @@ public function initialize(sfEventDispatcher $dispatcher, $parameters = array(),
     parent::initialize($dispatcher, $parameters, $attributes, $options);
 
     // GET parameters
-    $this->getParameters = get_magic_quotes_gpc() ? sfToolkit::stripslashesDeep($_GET) : $_GET;
+    if (version_compare(PHP_VERSION, '5.4', '<') && get_magic_quotes_gpc())
```

Comment:
I recommend to use `7.4.0-dev` instead of `7.4`.

>  Note that pre-release versions, such as 5.3.0-dev, are considered lower than their final release counterparts (like 5.3.0). 

source https://www.php.net/manual/en/function.version-compare.php

Reply:
@alquerci you mean `5.4.0-dev`, right? :)

Reply:
Ha, yes.

I make it as this patch is related to PHP 7.4 support. :)

Reply:
Thank you for the hint. Adjusted to `5.4.0-dev`